go.mod

```diff
@@ -120,5 +120,6 @@ require (
 replace (
 	github.com/Sirupsen/logrus => github.com/sirupsen/logrus v1.0.5
 	github.com/opencontainers/image-tools => github.com/sylabs/image-tools v0.0.0-20181006203805-2814f4980568
+	github.com/sylabs/scs-library-client => github.com/dctrud/scs-library-client v0.3.1-0.20190717221055-79bcf0162d6b
```

Comment:
I don't think we want to commit this, right?

Reply:
This is a WIP PR - It has to be tested against non-merged scs-library-client and a test library version, so I'm putting this here so it works for testing. It'll be a proper tagged scs-library-client before the WIP comes off.